util/reflect_test.go

```diff
@@ -771,31 +849,64 @@ func TestForEachField(t *testing.T) {
 }
 
 func TestUpdateFieldUsingForEachField(t *testing.T) {
+	containerSchema := &yang.Entry{
```

Comment:
Can this schema be made general for the file such that each test function doesn't need to define the same schema and the associated types that represent its generated code?

Reply:
Done. I've put it above the tests to keep some locality, otherwise the tests become very hard to understand.